### ISSUE TITLE
default process_exporter_names

### ISSUE THREAD

**Issue by gajowi** [cloudalchemy/ansible-process_exporter#10]
For consideration.
While the default variable is empty, there is no config.yml file so no test coverage of that functionality.
We *could* follow the project documentation and set a non-empty default

The default config shipped with the deb/rpm packages is:

```
process_names:
  - name: "{{.Comm}}"
    cmdline:
    - '.+'

```

Or not.  I think it is an open design decision and don't mind either way.

In any case, this issue is now noted and is discoverable if related issues come up in future.

An alternative approach is to tweak the CI test env. Perhaps a non-empty default is simpler. Doing nothing is even simpler :-)

**Comment by paulfantom** [comment]
Usually we follow both paths at the same time in 2 separate test scenarios. First one is testing default settings without changing any variables and second one tries to change as many as possible. Both scenarios should give a working setup.

I would suggest to start with a default list which is also shipped with deb/rpm packages.

For a reference design in setting up test scenarios you can go to https://github.com/cloudalchemy/ansible-prometheus/tree/master/molecule

**Comment by gajowi** [comment]
OK. I'll read more and work up a PR.

**Comment by gajowi** [comment]
closed by applying #11 - which lists more detail
